PhysicsTools/PatAlgos/python/slimming/customizeMiniAOD_HcalFixLegacy2016.py

```diff
@@ -15,24 +24,26 @@ def loadJetMETBTag(process):
     task.add(process.ak4PFJets)
     process.ak8PFJetsCHS = RecoJets.Configuration.RecoPFJets_cff.ak8PFJetsCHS.clone()
     task.add(process.ak8PFJetsCHS)
+
+    process.fixedGridRhoAll = RecoJets.Configuration.RecoPFJets_cff.fixedGridRhoAll.clone()
+    process.fixedGridRhoFastjetAll = RecoJets.Configuration.RecoPFJets_cff.fixedGridRhoFastjetAll.clone()
+    process.fixedGridRhoFastjetCentral = RecoJets.Configuration.RecoPFJets_cff.fixedGridRhoFastjetCentral.clone()
+    process.fixedGridRhoFastjetCentralChargedPileUp = RecoJets.Configuration.RecoPFJets_cff.fixedGridRhoFastjetCentralChargedPileUp.clone()
+    process.fixedGridRhoFastjetCentralNeutral = RecoJets.Configuration.RecoPFJets_cff.fixedGridRhoFastjetCentralNeutral.clone()
+    task.add( process.fixedGridRhoAll,
+              process.fixedGridRhoFastjetAll,
+              process.fixedGridRhoFastjetCentral,
+              process.fixedGridRhoFastjetCentralChargedPileUp,
+              process.fixedGridRhoFastjetCentralNeutral )
```

Comment:
these updates are making this method even less maintainable.
A practical solution would be to have a proper piece of jet reco task defined in RecoJets/Configuration and then used explicitly in both the standard reco and here.

My comment from May 1st still stands, https://github.com/cms-sw/cmssw/pull/22941#discussion_r185241096
it would be much better to load all configs in proper order as they appear in RECO+miniAOD jobs.
This workflow is done under run2_miniAOD_80XLegacy modifier. So, making these changes to load reco parts in proper order should not be difficult.